.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,29 @@
+name: Bug Report
+description: File a bug report
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        Please describe what you are trying to achieve and what went wrong.
+        Please include the stack trace of the error (error message) you are
```

Comment:
Perhaps put `error message' first and `including the stack trace if available` or similar in parens. To simplify the message for less tech savvy.

Reply:
Indeed!

Reply:
Done.